polyfill/lib/calendar.mjs

```diff
@@ -41,20 +41,14 @@ const impl = {};
 
 export class Calendar {
   constructor(id) {
-    // Note: if the argument is not passed, IsBuiltinCalendar("undefined") will fail. This check
-    //       exists only to improve the error message.
-    if (arguments.length < 1) {
-      throw new RangeError('missing argument: id is required');
-    }
```

Comment:
I'd suggest keeping something similar to this code when you make this change in the production polyfill — confusion over the error message when passing no arguments to the constructor was a big issue in user tests of the browser playground, way back when. You might have to change the exception to TypeError to match what RequireString throws.

Agree with removing this from here, though, since this code is now primarily for verifying the spec text and the browser playground is less important.

Reply:
Given that `RequireString(id)` will throw a message like "Expected a string, not undefined", I think the message is may be clear enough for now. If we get user confusion in the prod polyfill we can always add it, but I'm sensitive to adding download size to the prod polyfill so will probably leave it out initially and wait for complaints.